src/services/dbgap_telemetry_report.py

```diff
@@ -23,28 +23,38 @@ def call_telemetry_report(self):
 
         return xmltodict.parse(response.text)
 
-    def get_sample_status(self, alias, data_type):
+    def _get_sample(alias):
```

Comment:
Looks like `self` needs to be added back, and `call_telemetry_report` should be `self._ call_telemetry_report()`

```
    def _get_sample(self, alias):
        try:
            telemetry_data = self._call_telemetry_report()
```